test/correctness/simd_op_check.cpp

```diff
@@ -1455,23 +1460,23 @@ int main(int argc, char **argv) {
         in_i64 = ImageParam(Int(64), 1, "in_i64"),
         in_u64 = ImageParam(UInt(64), 1, "in_u64")
     };
-
-    for (ImageParam p : image_params) {
+    if (can_run_code()) {
```

Comment:
I suppose in the end it doesn't matter that much, because the only time an actual buffer is needed is to JIT and run the test later. So I think what you have is OK, it just needs a comment explaining this.